webui/react/src/Dev.ts

```diff
@@ -1,6 +1,15 @@
 /* Tools and tweaks for dev environments */
 
 import * as Api from 'services/api';
+import * as DetSwagger from 'services/api-ts-sdk';
+import { consumeStream } from 'services/apiBuilder';
+
+// TODO remove
+consumeStream<DetSwagger.V1TrialLogsResponse>(
+  DetSwagger.ExperimentsApiFetchParamCreator().determinedTrialLogs(1),
+  console.log,
+  () => console.log('finished'),
+);
```

Comment:
suggestion: I guess this is just a sample call. Maybe put it as a multiline comment above `consumeStream` function definition itself down below?

Reply:
I was planning to remove it but I can do that too.